public/app/features/browse-dashboards/BrowseDashboardsPage.tsx

```diff
@@ -39,6 +40,7 @@ const BrowseDashboardsPage = memo(({ match }: Props) => {
   const styles = useStyles2(getStyles);
   const [searchState, stateManager] = useSearchStateManager();
   const isSearching = stateManager.hasSearchFilters();
+  const location = useLocation();
```

Comment:
I'll have to look up what it is, but I think we have a 'preferred' way to listen to search params, maybe from locationSrv?

Otherwise, I think this apporach is good 👍

Reply:
Looks like we have a `LocationService`, returned from the the `useGrafana` hook.